### PR TITLE
Dynamic OG Images Module

### PR DESCRIPTION
**Pull request by bryantgillespie**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,3 @@
 DIRECTUS_URL="https://your-instance.directus.app"
+DIRECTUS_TOKEN="XHdfaklsdj;fladsjkfal;zj;lkda"
```
Comment (rijkvanzanten) [inline .env.example:2]: This is no longer relevant

File: modules/og-images/dist/og.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { defineComponent as r, openBlock as t, createElementBlock as s, createElementVNode as e, toDisplayString as o, createCommentVNode as l, createStaticVNode as g } from "vue";
```
Comment (rijkvanzanten) [inline modules/og-images/dist/og.js:1]: Lets not put bundled dist code in git

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
I think og-images/dist should be in gitignore right?

**Comment by bryantgillespie** [comment]
Closed in favor of #75

**Pull request closed without merging**